build/_deps/rippled-src/src/ripple/protocol/ErrorCodes.h

```diff
@@ -0,0 +1,343 @@
+//------------------------------------------------------------------------------
```

Comment:
This is a file from `rippled` that is part of the `build` directory and will be replaced on a clean build of `clio`.
Nothing in the `build` directory should be ever added to the repo so unfortunately this is not a viable solution.
If we were to add this error code we would have to add it in the `rippled` repo.

Perhaps it's time for `clio` to rethink how we handle custom error codes.